dts/arm/st/f0/stm32f0.dtsi

```diff
@@ -358,6 +359,13 @@
 		io-channels = <&adc1 17>;
 		status = "disabled";
 	};
+
+	vbat: vbat {
```

Comment:
Since STM32F0x0 don't have Vbat channel, you should add this node in stm32f031.dtsi. All non-F0x0 dtsi include this file. 
This would remove the need to delete the node for F030 and F070.
This is also true for the vbat-channel property.

Reply:
Oh I didn't see that, good catch, thanks! I'll amend the commit.